scripts/landing.js

```diff
@@ -0,0 +1,25 @@
+var animatePoints = function() {
+  var points = document.getElementsByClassName('point');
+  var revealFirstPoint = function() {
+    points[0].style.opacity = 1;
+    points[0].style.transform = "scaleX(1) translateY(1rem)";
+    points[0].style.msTransform = "scaleX(1) translateY(1rem)";
+    points[0].style.WebkitTransform = "scaleX(1) translateY(1rem)";
+  };
+  var revealSecondPoint = function() {
+    points[1].style.opacity = 1;
+    points[1].style.transform = "scaleX(1) translateY(1rem)";
+    points[1].style.msTransform = "scaleX(1) translateY(1rem)";
+    points[1].style.WebkitTransform = "scaleX(1) translateY(1rem)";
+  };
+  var revealThirdPoint = function() {
+    points[2].style.opacity = 1;
+    points[2].style.transform = "scaleX(1) translateY(1rem)";
+    points[2].style.msTransform = "scaleX(1) translateY(1rem)";
+    points[2].style.WebkitTransform = "scaleX(1) translateY(1rem)";
+  };
+  revealFirstPoint();
+  revealSecondPoint();
+  revealThirdPoint();
+};
+animatePoints();  
```

Comment:
this isn't a direct copy of the js from the checkpoint instructions - i changed some of the values around to make sure i understand how everything works.